src/uct/ib/dc/dc_mlx5.c

```diff
@@ -1372,11 +1365,13 @@ static UCS_CLASS_INIT_FUNC(uct_dc_mlx5_iface_t, uct_md_h tl_md, uct_worker_h wor
     ucs_assert(self->tx.num_dci_pools <= UCT_DC_MLX5_IFACE_MAX_DCI_POOLS);
 
     /* allocate dcis arrays and stack */
-    self->tx.dcis = ucs_calloc(UCT_DC_MLX5_IFACE_MAX_DCIS(self->tx.ndci,
-                               self->tx.num_dci_pools), sizeof(*self->tx.dcis),
+    self->tx.dcis = ucs_calloc((self->tx.ndci * self->tx.num_dci_pools +
```

Comment:
( ) around * expression

Reply:
done

> also to check no perf degradation

always verified for each commit